src/components/Background.tsx

```diff
@@ -12,6 +12,7 @@ const Div = styled.div<{ background: Background }>`
   }
   background-repeat: repeat-y;
   background-size: cover;
+  height: 100%;
```

Comment:
Swwweet! Approved with one small change.

I just noticed something of my own omission. 

We don't need `height: 100%` here because this overrides the purpose of `background-repeat`. You can see what happens here:

<img width="1438" alt="Screenshot 2020-10-26 at 15 21 35" src="https://user-images.githubusercontent.com/53219789/97185097-f609ac00-179f-11eb-9b01-585ceaef4f3d.png">

What we actually want is for the background to repeat _sometimes_ so we can get rid of this line.

Reply:
actually no ignore that - this doesn't work. I need to look at this again

Reply:
nope, sorry. I've found myself in the dreaded CSS vortex where one thing works but breaks something else. 

can we phone a friend? aka @mantagen